balancer-js/src/lib/utils/math.ts

```diff
@@ -28,3 +28,6 @@ export function parseToBigInt18(value: string): bigint {
 export function formatFromBigInt18(value: bigint): string {
   return _formatFixed(BigNumber.from(value), 18);
 }
+
+export const bn = (value: number): BigNumber => _parseFixed(`${value}`, 18);
```

Comment:
Do you think we should call these something more descriptive? Like `parseFixedFromNumber`?

Reply:
Do you mean `bn`? I took the name from monorepo, just because I liked how short it was. Do you think we should stick to a better standard here?

Reply:
I agree with John. I believe `bn` does not describe what's going on. `parseFixedFromNumber` is indeed more descriptive, but in this case I believe I'd stick with actually not replacing parseFixed at all.
Considering the use case of `bn(1)`, I'd rather read `parseFixed('1', 18)` because then I know exactly what's being done.
Regarding `fp` below, if I understood correctly, it's trying to make it easier to parse float points, right? In that case I'd go with `parseFloat` or something like that.

Reply:
That's funny because I see it in a completely opposite way. I think there is a lot of benefit to have short names when you use them frequently. Converting string to BigNumber and back is so common that we could use a simple short name. However if you find `parseFixedFromNumber` useful, let's add it as well! Should I add an alias?

Reply:
I think I find something like `parseFixedFromNumber` useful but can understand the argument for shorter. So maybe an alias is good compromise.

Reply:
If that's something that will be used very frequently, I can see the benefit of having short names. For example, we always use `bpt` instead of `balancerPoolToken` and that's totally fine.
I think my concern comes from the fact that it's the first time I've seen `bn` or `fp` in the code.
If we start to use both more often and make them a standard, then I'd be ok with that.
Btw, what does `fp` stand for? I thought it was `floatPoint`, but it does not result in a float point, so I'm not sure I got it right 😅

Reply:
Based on the implementation `fp` looks more like a `fixedPoint` - again, I feel that kind of confusion could/should be avoided 😬

Reply:
true, let's remove that.